grishma/tests/acceptance/features/login.feature

```diff
@@ -0,0 +1,10 @@
+Feature: login
+As a user
+I want to do login
+So I can access my profile
```

Comment:
```suggestion
Feature: login
   As a user
   I want to be able to log into my account
   So that i have access to my files
```
isn't the AUT is regarding the file system?